docker/queue-monitor/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# echo "Starting VNC server..."
+# # Setup VNC server
+# Xvfb :1 -screen 0 2560x1440x16 &
+# x11vnc -rfbport 5901 -bg -quiet -forever -shared -display :1
+
+echo "Starting queue-monitor GUI..."
+echo "Control address: $QSERVER_ZMQ_CONTROL_ADDRESS"
+echo "Publish address: $QSERVER_ZMQ_INFO_ADDRESS"
+queue-monitor
```

Comment:
I think will spawn `queue-monitor` each time a container command is run from the host.  Should this be a `CMD` instead?

Reply:
Good thought, I've updated to use CMD.